config/wc-config.yaml

```
@@ -298,6 +298,51 @@ prometheusBlackboxExporter:
 
 ingressNginx:
   subDomain: ingress-nginx
+  controller:
+    ## Kubernetes service configuration.
+    service:
+      internal:
+
+        ## Type of service.
+        ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types
+        type: set-me-if-(.ingressNginx.controller.service.enabled)
```

I assume we want this for the internal service
```suggestion
        type: set-me-if-(.ingressNginx.controller.service.internal.enabled)
```